waiter/bin/ci/run-integration-tests-k8s-scheduler.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+# Usage: run-integration-tests-k8s-scheduler.sh [TEST_COMMAND] [TEST_SELECTOR]
+#
+# Examples:
+#   run-integration-tests-k8s-scheduler.sh parallel-test integration-fast
+#   run-integration-tests-k8s-scheduler.sh parallel-test integration-slow
+#   run-integration-tests-k8s-scheduler.sh parallel-test
+#   run-integration-tests-k8s-scheduler.sh
+#
+# Runs the Waiter integration tests using the (local) k8s scheduler, and dumps log files if the tests fail.
+
+set -e
+
+TEST_COMMAND=${1:-parallel-test}
+TEST_SELECTOR=${2:-integration}
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+WAITER_DIR=${DIR}/../..
+KITCHEN_DIR=${WAITER_DIR}/../kitchen
+
+# Start minikube
+${DIR}/minikube-setup.sh
+
+# Pre-fetch our test image
+docker pull daowen/waiter-kitchen:latest
```

Comment:
I don't understand your question. Are you talking about the docker image namespace? Or are you talking about how we can pre-fetch container images internally at 2σ?

Reply:
I am talking about the docket image namespace, e.g. `two-sigma/waiter-kitchen:latest`

Reply:
The namespace indicates the DockerHub account used when pushing the image. If we want to use a different DockerHub account, we can do that, but I didn't think this namespace matters that much since we pull several other dependencies from non-2σ Clojars and DockerHub namespaces.

Reply:
❌

Reply:
We could build the image in Travis (just like we do with minimesos), which would ensure we have the right kitchen. Maybe I can reuse the same image. I'll check.